presto-geospatial/src/main/java/com/facebook/presto/plugin/geospatial/GeometryType.java

```diff
@@ -83,6 +86,11 @@ public Object getObjectValue(ConnectorSession session, Block block, int position
             return null;
         }
         Slice slice = block.getSlice(position, 0, block.getSliceLength(position));
-        return deserialize(slice).asText();
+        try {
```

Comment:
Is this error path tested exercised by any test? Looks like deserialize catches the GeometryException and wraps it so is this necessary?